packages/core/admin/admin/src/components/AutoReloadOverlayBlocker/Wrapper.js

```diff
@@ -0,0 +1,94 @@
+import styled, { keyframes } from 'styled-components';
+
+// TODO refacto when DS ready
```

Comment:
Same question here, how the DS will solve this? Is there any component related to the Overlayblocker or maybe you are talking about the theme?

Reply:
Well with the new DS there will be a new design for this component and this is when we should refactor 
both the CSS and the related components.